README.md

```diff
@@ -10,3 +10,20 @@ To build a new Solidus Extension, you can run
 solidus extension my_extension
 ```
 
+after you have globally installed the gem outside of your project folder:
+
+```
+gem install solidus_cmd
+```
+
+This will create a folder `solidus_my_extension` containing default content to
+let you start writing your own extension.
+
+Please update the .gemspec file with author, description, summary and email.
```

Comment:
There is more files than the gemspec that need to be updated by the user, and that files can change in the future, so I don't think that we should reference each file that need to be updated.

I think we can just mention that they need to update generated files.

Reply:
Maybe it would be useful to have a list of files that contain generic stuff, like the LICENSE too, to avoid that the user should check each file to see if there is something to fill in.